packages/block-library/src/gallery/edit.js

```diff
@@ -117,13 +117,13 @@ export default function GalleryEdit( props ) {
 		onFocus,
 	} = props;
 
-	const lightboxSetting = useSettings( 'blocks.core/image.lightbox' )[ 0 ];
+	const [ lightboxSetting ] = useSettings( 'blocks.core/image.lightbox' );
```

Comment:
Unrelated nit: I think the array destruction is easier to read here.

Reply:
True, thanks for the drive-by improvement 👍